tsconfig.json

```diff
@@ -0,0 +1,81 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                         /* Enable incremental compilation */
+    "target": "ES2020",                                /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
+    "module": "commonjs",                           /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
+    // "lib": [],                                   /* Specify library files to be included in the compilation. */
+    "allowJs": true,                             /* Allow javascript files to be compiled. */
+    // "checkJs": true,                             /* Report errors in .js files. */
```

Comment:
Would this be helpful to have enabled?

Reply:
It would be, but requires a lot more changes across the application to fix bad code and those changes are out of scope for this feature. I'd recommend creating an issue with the sole focus of enabling `checkJs`. You'll likely need to turn off `noImplicitAny` in order to avoid a shotgun surgery, and we don't want implicit `any` anywhere in our code so disabling that check is dangerous. The other option is turning off `strict` but that's even worse.